### DISCUSSION TITLE
Using Probability Integral Transforms with Bayesian Regression

### DISCUSSION THREAD

**Discussion by shuvayan** [pymc-labs/pymc-marketing#700] [Ideas]
Hello ,

I have been reading a research paper which talks about using PIT for Robyn results validations. I was wondering if the same is applicable fot bayesian structure as the frequentist approach of some methods elicitate statistical validations.
However, I think since in bayesian we are restricting the variance of the parameters via priors- such tests might not be neccessary/useful at all.

I would like to know the views and opinions of the forum and dispel any wrong notions I might be having.
In case it is applicable and useful, where can we start building this into this awesome library?

**Comment by williambdean** [comment]
I don't know much about this. Do you have an references and links?
